components/alertbanner/stories/alertbanner.stories.js

```diff
@@ -1,14 +1,19 @@
 import { html } from "lit";
+import { styleMap } from "lit/directives/style-map.js";
+
+import { Template as Typography } from "@spectrum-css/typography/stories/template.js";
 import { Template } from "./template";
 
 export default {
 	title: "Components/Alert banner",
 	description:
 		"The Alert banner show pressing and high-signal messages, such as system alerts. They’re meant to be noticed and prompt users to take action.",
 	component: "AlertBanner",
+	subtitle: "Testing",
 	argTypes: {
 		isOpen: {
 			name: "Open",
+			description: "Determines whether the banner is visible or hidden.",
```

Comment:
It's interesting that the Alert banner is `display: none` by default, and only displays if the `is-open` class is present. Do you think it would be worth keeping the control help make sure this is clear? How about adding an example to the Chromatic-only template without the `is-open` class, to make sure that it remains `display:none` in VRTs?

Reply:
I've added that hidden alert banner to the stories (in the code you'll see  _"This alert banner should never be displayed in VRTs..."_).